docs/animations/Expressions.md

```diff
@@ -0,0 +1,110 @@
+# ExpressionBuilder 
+Welcome to the ExpressionBuilder classes! The ExpressionBuilder classes are a C#-only alternative to building Expressions with type safety. Below is a quick introduction to using the ExpressionBuilder classes with your application. For complete documentation and walkthroughs, please see the Word Document in the project folder titled "ExpressionBuilder_Documentation.docx".
```

Comment:
I tried to bring it over with an online tool, but all the code snippets are images for some reason. For this release, let's link to the doc, and then we can bring it in with the rest of the documentation after 1.5